lib/spack/spack/cmd/buildcache.py

```diff
@@ -443,9 +440,12 @@ def createtarball(args):
     add_spec = ('package' in args.things_to_install)
     add_deps = ('dependencies' in args.things_to_install)
 
-    _createtarball(env, args.spec_yaml, args.specs, add_spec, add_deps,
-                   output_location, args.key, args.force, args.rel,
-                   args.unsigned, args.allow_root, args.no_rebuild_index)
+    _createtarball(env, spec_yaml=args.spec_yaml, packges=args.specs,
```

Comment:
```suggestion
    _createtarball(env, spec_yaml=args.spec_yaml, packages=args.specs,
```

Reply:
Ah for pete's sake.  Good catch, thanks.